tensorrt_llm/_torch/auto_deploy/transformations/library/sharding.py

```diff
@@ -85,6 +92,15 @@ def _insert_sharded_matmul(
     def split_tensor(
         t: torch.Tensor, d: int = dim, r: int = rank, ws: int = world_size
     ) -> torch.Tensor:
+        # The local tensor shape has to be divisible by min_local_shape
+        max_split_size = t.shape[d] // min_local_shape
+        if ws > max_split_size:
+            num_groups = math.ceil(ws / max_split_size)
+            ad_logger.warning(
```

Comment:
```suggestion
            ad_logger.debug(
```
Looks more like a case for debug log than a warning. This behavior is unaccounted for - so nothing to warn about